poetry.lock

```diff
@@ -1752,23 +1774,23 @@ extra = ["pygments (>=2.12)"]
 
 [[package]]
 name = "pyspark"
-version = "3.4.1"
+version = "3.5.1"
```

Comment:
@amaldevk88 Please pull the latest version of the code base. The unit tests are failing because of the pyspark version and it is pinned down to <3.5 now.

Reply:
changes are re-pushed. Please run the checks

Reply:
Please review.